src/main/java/com/example/spring_bank_project/Account/Infrastructure/Gateway/Http/Controller/User/UserController.java

```diff
@@ -0,0 +1,20 @@
+package com.example.spring_bank_project.Account.Infrastructure.Gateway.Http.Controller.User;
+
+import com.example.spring_bank_project.Account.Application.UserFacade;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(name = "api.user.", path = "/api/user")
+public class UserController {
+    private final UserFacade facade;
+
+    @PostMapping(path = "/store", name = "store")
```

Comment:
powinienes zwracac zawsze jakis repsponse z kontolera, nawet jesli ma to byc void to zwroc ResponseEntity.ok().build() a w definicji metody daj ResponseEntity<Void>

Reply:
name chyba tez zbedne